class.jetpack-post-images.php

```diff
@@ -85,6 +90,11 @@ static function from_gallery( $post_id ) {
 		$images = array();
 
 		$post = get_post( $post_id );
+
+		if ( ! $post ) {
+			return;
```

Comment:
Elsewhere in this function `return $images` (which is set at the beginning as an empty array) is used and in the other hunk, `return false` is used.

Should we be consistent here with what type we're returning?